engine/engine_launcher.go

```diff
@@ -73,7 +72,7 @@ type Launcher struct {
 
 func NewEngineLauncher(spec *rpc.EngineSpec) (*Launcher, *Engine) {
 	el := &Launcher{
-		UUID:         uuid.NewV4().String(),
+		UUID:         spec.Uuid,
```

Comment:
In fact it should be spelled as `UUID` since it's short for universally unique identifier

Reply:
This name is generated by protoc. If we rename `uuid` to `u_u_i_d` in the pb file, we can get `UUID`